server/src/helpers/translation.js

```diff
@@ -47,12 +49,38 @@ const getTranslatedText = async (text, originalLanguage) => {
   }
 }
 
+const readTranslationMockFile = async file => {
+  try {
+    const rawData = fs.readFileSync(file)
```

Comment:
`readFileSync` bloque l'exécution de node pendant la lecture du fichier, autrement dit aucune requête HTTP entrante ne pourra être traitée pendant ce temps. Utilise plutôt `const fs = require('fs/promises')` en haut puis ici 
```suggestion
    const rawData = await fs.readFile(file)
```
Pour rendre la lecture du fichier asynchrone.